contracts/impls/trustable/WitnetRequestBoardV03OVM.sol

```diff
@@ -1,129 +1,116 @@
 // SPDX-License-Identifier: MIT
 
+/* solhint-disable var-name-mixedcase */
+
 pragma solidity >=0.7.0 <0.9.0;
 pragma experimental ABIEncoderV2;
 
 // Inherits from:
-import "../WitnetRequestBoardUpgradableBase.sol";
+import "../WitnetBoardUpgradableBase.sol";
 import "../../data/WitnetBoardDataACLs.sol";
 
 // Uses:
 import "../../interfaces/IERC20.sol";
 
-/**
- * @title Witnet Requests Board V03 - OVM 
- * @notice Contract to bridge requests to Witnet Decenetralized Oracle Network.
- * @dev This contract enables posting requests that Witnet bridges will insert into the Witnet network.
- * The result of the requests will be posted back to this contract by the bridge nodes too.
- * @author Witnet Foundation
- */
+/// @title Witnet Requests Board V03 - OVM.
+/// @notice Contract to bridge requests to Witnet Decenetralized Oracle Network.
```

Comment:
```suggestion
/// @notice Contract to bridge requests to Witnet Decentralized Oracle Network.
```

Reply:
This change is postponed to `0.4.x` as it would imply breaking changes on the price feed examples, for instance.